src/com/koxudaxi/poetry/PyAddNewPoetryPanel.kt

```diff
@@ -173,20 +174,31 @@ class PyAddNewPoetryPanel(private val project: Project?,
         }
     }
 
+    private val isPoetry by lazy { existingSdks.filter { it.isPoetry }.map { it.associatedModulePath to it }.toMap() }
```

Comment:
`associateBy` is better a way.

`.map { it.associatedModulePath to it }.toMap() }`
↓
`.associateBy { it.associatedModulePath } }`

Reply:
Done, thank you for the tip!